src/ducks/i18n.js

```diff
@@ -0,0 +1,26 @@
+import i18n from 'i18next';
```

Comment:
as this file is not related to reducers./redux . please remove that from here and keep it in src folder level